tests/PHPStan/Analyser/nsrt/bug-12393.php

```diff
@@ -143,3 +143,23 @@ public function getMixed()
 	}
 
 }
+
+// https://3v4l.org/LK6Rh
+class CallableString {
+	private string $foo;
+
+	public function doFoo(callable $foo): void {
+		$this->foo = $foo; // PHPStorm wrongly reports an error on this line
```

Comment:
opened a PHPStorm bug: https://youtrack.jetbrains.com/issue/WI-81263/Property-must-be-of-type-string-callable-provided-false-positive